iam-login-service/src/main/java/it/infn/mw/iam/notification/NotificationFactory.java

```diff
@@ -39,4 +41,9 @@ public interface NotificationFactory {
   IamEmailNotification createGroupMembershipApprovedMessage(IamGroupRequest groupRequest);
 
   IamEmailNotification createGroupMembershipRejectedMessage(IamGroupRequest groupRequest);
+
+  IamEmailNotification createClientStatusChangedMessage(ClientDetailsEntity client);
+
```

Comment:
I'd prefer (as suggested above) a unique method that receives client object (owners and contacts can be retrieved from it)